### PR TITLE
delete page now shows again

### PR DESCRIPTION
**Pull request by m-hakkestad**

### CODE REVIEW

File: src/components/delete/DeleteForm.js [hunk 1]
```diff
@@ -28,16 +28,16 @@ class DeleteForm extends React.Component {
 
     render() {
         return (
-            <form className="white-form">
+            <form className="delete-Form">
```
Comment (codebyjonas) [inline src/components/delete/DeleteForm.js]: Not `"` should be `'`, goes for the enitre file.

File: src/components/delete/DeleteForm.js [hunk 2]
```diff
@@ -28,16 +28,16 @@ class DeleteForm extends React.Component {
...
                 <input
                     onChange={(e) => this.setState({ password: e.target.value })}
                     placeholder="Confirm Password"
                     type="password" className="delete-Input">
                 </input>
                 <div className="delete-Options">
-                    <button className="white-button" onClick={this.deleteAccount}>Confirm</button>
-                    <NavLink to='/settings' className="white-button" > Gosh no, cancel </NavLink>
+                    <button className="delete-Button" onClick={this.deleteAccount}>Confirm</button>
+                    <Link to='/settings' className="delete-Button" > Gosh no, cancel </Link>
```
Comment (codebyjonas) [inline src/components/delete/DeleteForm.js]: rename `delete-button`

File: src/index.js [hunk 3]
```diff
@@ -71,6 +72,7 @@ class Index extends React.Component {
                       <Route exact path='/feed' render={(props) => <Feed {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
                       <Route exact path='/search' render={(props) => <Search {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
                       <Route exact path='/settings' render={(props) => <Settings {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
+                      <Route exact path='/delete' render={(props) => <Delete {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
                       <Route exact path='/login' component={() => <Settings updateUserId={this.updateUserId} userId={this.state.userId} />} />
                       <Route exact path='/register' render={(props) => <Register {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
                       <Route path='/modify-user' render={(props) => <ModifyUser {...props} updateUserId={this.updateUserId} userId={this.state.userId} />} />
```
Comment (codebyjonas) [inline src/index.js]: Should also have the attribute`exact`

File: src/pages/delete.js [hunk 4]
```diff
@@ -1,69 +1,69 @@
...
+            <Header/>
+            <main className="delete">
+                {objToRender}
+            </main>
+            <Footer/>
+        </div>
+    )
+}
 }
 export default Delete
```
Comment (codebyjonas) [inline src/pages/delete.js]: Run a autoformat on this file

File: src/styles/delete.scss [hunk 5]
```diff
@@ -1,5 +1,71 @@
...
+            font-size: 1.3rem;
+            border-radius: $borderRadius;
+
+            &::placeholder {
+              color: #ffe6e6;
+              font-size: 1.3rem;
+            }
+        }
+
+        .delete-Options{
```
Comment (codebyjonas) [inline src/styles/delete.scss]: Use the css in the file _standards-element.scss_

**Comment by m-hakkestad** [comment]
Changes fixed!

**Pull request merged**